src/os/bluestore/BlueStore.cc

```diff
@@ -6881,6 +6890,16 @@ int BlueStore::mkfs()
     goto out_close_bdev;
   }
 
+  // make sure min_alloc_size is >= and aligned with block size
+  if (min_alloc_size % block_size != 0) {
+    derr << __func__ << " min_alloc_size 0x"
+	 << std::hex << min_alloc_size << std::dec
+	 << " is less or not aligned with block_size: 0x"
+	 << std::hex << block_size << dendl;
```

Comment:
move the first std::dec to the end and remove the second std::hex?

Reply:
Done